website/docs/reference/resource-limits.mdx

```diff
@@ -38,6 +38,20 @@ Existing hosted customers who are already above the limits will have their limit
 
 With a few exceptions, all the limits can be overridden through the use of their respective environment variables. However, while it is possible to override these limits, we advise against it in most cases. If you feel that you're unable to do what you need to do because of the limits, we'd love to hear from you.
 
+Some of the limits have a lower threshold. If you try to set them lower than that threshold, Unleash will instead use the minimum value listed below:
+
+| Resource          | Minimum value |
+|-------------------|--------------:|
+| Feature flags     |             1 |
+| Strategies        |             1 |
+| Constraints       |             0 |
```

Comment:
I don't get this one. What is the difference between having no minimum value or having a minimum value of 0? We can't have a negative amount of API keys, so not sure what the implications would be?

Reply:
Practically? Probably nothing, but I haven't checked what happens if you try to set segment values limit to -42. 

As for why some numbers are 0 and some are 1 in this table, that's because we decided when we set them up, that you might want to run Unleash without allowing people to use constraints, segments, or API tokens. But if you disallow flags or strategies entirely (i.e. set them to 0), then Unleash can't do anything. Unleash also needs at least 1 project and 1 environment to work. And finally, if you do activate constraints, then it doesn't make sense to not allow any values for them.

Reply:
That said, I have a little bit of time now. I'll see what happens

Reply:
Yeah, I'm guessing that this is what's gonna happen for all the other resources that don't have set limits:

![image](https://github.com/user-attachments/assets/8019c161-e309-41a1-a385-f88a6efe9325)

I'd be happy to mention those with a 0 too?

Reply:
So from a conceptual point of view, wouldn't it make sense to just mention the ones that *do* have a limit of 1? and leave the rest out (if the 0 limit is basically the same in practice as no limit)

So something like:

The following resources have a lower limit of 1:
- feature flags
- strategies
- constraints values
- projects
- environments

If you set any of these values below the minimum, Unleash automatically uses a value of 1 instead.

Reply:
Yeah, that's a good way to do it. I'll have a look.